packages/html-reporter/playwright.config.ts

```diff
@@ -27,6 +27,7 @@ const config: PlaywrightTestConfig = {
     ['html', { open: 'on-failure' }]
   ],
   use: {
+    baseURL: 'file://' + path.join(__dirname, 'out-ct', 'index.html').replace(path.sep, '/'),
```

Comment:
```suggestion
    baseURL: 'file://' + path.join(__dirname, 'out-ct', 'index.html').replace(path.sep, '/'),
```

-> url.pathToFileURL()